### PR TITLE
Support additional `rel` values on `Link` and `ThemedLink`

### PR DESCRIPTION
**Pull request by fow5040**
We wanted the ability to define `rel` attribute values on the `ThemedLink` component (specifically, `rel='nofollow'`) for SEO purposes. 

It looks like there is already some default `rel` definition behavior in the UIAction directory, mainly for anchor elements - the intent of this PR is that we do not affect any existing rel definition and instead add any new definitions (without duplication) to the existing definitions.

### CODE REVIEW

File: packages/thumbprint-react/components/Link/index.tsx [hunk 1]
```diff
@@ -104,7 +104,7 @@ interface LinkPropTypes {
      */
     onClick?: () => void;
     /**
-     * The anchor `rel` attribute. Setting this value will override any default value provided by
+     * The anchor `rel` attribute. Setting this value will add to any default values provided by
      * thumbprint for the `rel` attribute.
```
[the remark this answers is not in the document]
Comment (fow5040) [reply packages/thumbprint-react/components/Link/index.tsx]: Done - rebased and pushed to this branch!

**Comment by egoens** [comment]
@fow5040 was there any particular incident or request that is driving this request? I see that you mentioned that this was for SEO purposes, but would you be able to add a bit more color to that statement?

**Comment by fow5040** [comment]
Sure - we have static links (anchors) to dynamic pages (i.e. Instant Results page) that we do not want to show up in search results, but we still want to use the links rather than convert them all to be dynamic buttons. We have some guards to prevent Google from crawling these links, but were advised by an SEO consultancy to add the 'nofollow' rel to ensure Google does not attempt to navigate through these links and increase our SEO rankins.

**Review by danoc — APPROVED** [review]
Thanks for this, @fow5040! I'll merge and release it. 🙇🏻

**Pull request merged**